dotcom-rendering/src/components/StickyBottomBanner.importable.tsx

```diff
@@ -59,14 +60,8 @@ type RRBannerConfig = {
 };
 
 const getBannerLastClosedAt = (key: string): string | undefined => {
-	// eslint-disable-next-line no-restricted-syntax -- FIXME-libs-storage
-	const item = localStorage.getItem(`gu.prefs.${key}`) as undefined | string;
-
-	if (item) {
-		const parsedItem = JSON.parse(item) as Record<string, any>;
-		return parsedItem.value;
-	}
-	return item;
+	const item = storage.local.get(`gu.prefs.${key}`);
+	return isString(item) ? item : undefined;
```

Comment:
Just double-checking / leaving-a-note that this refactor is a no-op in regards to the `.value` lookup because [it happens in `libs/storage`](https://github.com/guardian/csnx/blob/main/libs/%40guardian/libs/src/storage/storage.ts#L36).